browser/src/layer/tile/CommentListSection.ts

```diff
@@ -1588,8 +1588,12 @@ export class CommentSection extends CanvasSectionObject {
 		}
 
 		lastY += this.containerObject.getDocumentTopLeft()[1];
-		if (lastY > app.file.size.pixels[1])
-			app.view.size.pixels[1] = lastY;
+		if (lastY > app.file.size.pixels[1]) {
+			if (app.view.size.pixels[1] !== lastY) {
+				app.view.size.pixels[1] = lastY;
+				this.onResize(); // Annotation goes beyond document and can't be scrolled further unless resized
+			  }
+		}
```

Comment:
Revisiting this PR again. I still don't see a difference with this patch applied. Which is probably expected specially in the following case (because it doesn't account for the comment's height, maybe?) this is what I see with or without the patch:


https://github.com/CollaboraOnline/online/assets/65948705/d1cdfe53-919d-4e99-9a62-1b726014b297

And I might I have some oddity with view position and the scrollbar itself. The view seems not tall enough (can't scroll to see the whole comment) but if I scroll up first then I can scroll down further and see the whole comment:

https://github.com/CollaboraOnline/online/assets/65948705/aba04d0a-d3ec-43f6-ada1-c527e248f295

Reply:
This patch exactly solves that problem, to get a comment into the view you first had to scroll up and then down to see the comment, now you can directly scroll down it just extends the scrollable part :)